src/components/InputTodo.jsx

```diff
@@ -1,4 +1,36 @@
-const InputTodo = () => {
-  return <div>input field here...</div>;
+/* eslint-disable react/prop-types */
+import { useState } from 'react';
+
+const InputTodo = ({ addTodoItem }) => {
+  const [title, setTitle] = useState('');
+  const [message, setMessage] = useState('');
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    if (title.trim()) {
+      addTodoItem(title);
+      setTitle('');
+      setMessage('');
+    } else {
+      setMessage('Please add item.');
+    }
+  };
+
+  const handleChange = (e) => {
+    setTitle(e.target.value);
+  };
+
+  return (
+    <>
+      <form onSubmit={handleSubmit} className="form-container">
+        <input type="text" placeholder="Add Todo..." className="input-text" value={title} onChange={handleChange} />
+        <button type="button" className="input-submit">
+          Submit
+        </button>
```

Comment:
Items are added to your list only when the enter key is pressed. This therefore button serves no purpose. It's better your convert it to a simple text so that users don't bother displacing the arrow to click a button than won't do anything